### PR TITLE
fix deprecations in Julia 0.7

### PR DESCRIPTION
**Pull request by heliosdrm**
Fixed deprecations observed in test for Jula 0.7.0-beta

### CODE REVIEW

**Comment by heliosdrm** [comment]
Wait. Breaking change for 0.6, to be fixed.

**Comment by KristofferC** [comment]
Personally, I think we can just upgrade to 0.7 now. Don't see much point releasing a new 0.6 release with only deprecation fixes for 0.7 in it.

File: REQUIRE [hunk 1]
```diff
@@ -1,2 +1,2 @@
-julia 0.6
-Compat 0.62.0
+julia 0.7
+Compat 0.54.0
```
Comment (KristofferC) [inline REQUIRE]: Can we just remove compat then?

**Comment by heliosdrm** [comment]
Updated the PR to break 0.6 support and test only against 0.7 (+ nightly).

File: REQUIRE (hunk 1 above)
Comment (heliosdrm) [reply REQUIRE]: Yes, unless there is some other compat function or macro in the previous versions (I didn't check that).

Comment (KristofferC) [reply REQUIRE]: I think this has to be `julia 0.7-` to show that it is for a prerelease of 0.7.

**Pull request merged**